### ISSUE TITLE
Call to undefined function dbFetchRows() in /opt/librenms/includes/polling/os/vmware-esxi.inc.php:15

### ISSUE THREAD

**Issue by trcyberoptic** [librenms/librenms#15383]
### The problem

Hi
after todays daily update cycle we get the following error on all VMware ESXi host polls:

> Error polling os module for pesx1. Error: Call to undefined function dbFetchRows() in /opt/librenms/includes/polling/os/vmware-esxi.inc.php:15
> Call to undefined function dbFetchRows() {"exception":"[object] (Error(code: 0): Call to undefined function dbFetchRows() at /opt/librenms/includes/polling/os/vmware-esxi.inc.php:15)"}
> 

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 23.9.1-42-gbe24993cb (2023-10-05T08:29:22+02:00)
DB Schema | 2023_08_30_105156_add_applications_soft_deleted (259)
PHP       | 8.1.23
Python    | 3.11.2
Database  | MariaDB 10.11.3-MariaDB-1
RRDTool   | 1.7.2
SNMP      | 5.9.3
===========================================

[OK]    Composer Version: 2.6.4
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQl and PHP time match
[OK]    Active pollers found
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[OK]    Python poller wrapper is polling
[OK]    Redis is unavailable
[OK]    rrdtool version ok
[OK]    Connected to rrdcached
```

### What was the last working version of LibreNMS?

the one before 23.9.1-42-gbe24993cb

**Comment by murrant** [comment]
#12567

**Comment by murrant** [comment]
#15008

**Comment by murrant** [comment]
Should be fixed now that #15008 is merged
